addOns/websocket/src/main/zapHomeFiles/scripts/scripts/websocketpassive/email_disclosure.js

```diff
@@ -0,0 +1,37 @@
+// * Author Manos Kirtas (Manos Kirtas)
+// * Based on: community-scripts/passive/Find Emails.js
+
+// Passive scan rules should not send messages
+// Right click the script in the Scripts tree and select "enable"  or "disable"
+
+OPCODE_TEXT = 0x1;
+RISK_INFO 	= 0;
+CONFIDENCE_HIGH = 3;
+
+var emailRegex = new RegExp("([a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+[a-zA-Z0-9-]*\.[a-zA-Z0-9-.]*[a-zA-Z0-9]{2,})", 'gmi');
+
+function scan(helper,msg) {
+
+    if(msg.opcode != OPCODE_TEXT || msg.isOutgoing){
+        return;
+    }
+
+    var counter = 0;
+    var message = String(msg.getReadablePayload()).valueOf();
+
+    if((matches = message.match(emailRegex)) != null) {
+        matches.forEach(function(evidence){
+            helper.newAlert()
+                .setRiskConfidence(RISK_INFO, CONFIDENCE_HIGH)
+                .setName("Email address found in WebSocket message (script).")
+                .setDescription("Email address were found at WebSocket Message.")
+                .setParam("Order: " + counter++)
```

Comment:
I think this is probably used on some handling in the [Find HTML Comments](https://github.com/zaproxy/community-scripts/blob/27efdf1cb39dee4f9dd71d7055e4b5a6ee231c42/passive/Find%20HTML%20Comments.js#L23-L24) script. However: 1) It's an older script. 2) It uses parameters in order to facilitate different grouping of results.

In the majority of cases passive scanners (scripts or other) should not really have/need a "parameter".

Reply:
Fixed in 30cc867